src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -411,102 +235,112 @@ public override void Save(ModelSaveContext ctx)
-            return _outputType;
+            Contracts.CheckValue(env, nameof(env));
+            var host = env.Register(RegistrationName);
+            host.CheckValue(ctx, nameof(ctx));
+            ctx.CheckAtModel(GetVersionInfo());
+            return new StopWordsRemoverTransform(host, ctx);
         }
 
-        protected override Delegate GetGetterCore(IChannel ch, IRow input, int iinfo, out Action disposer)
+        // Factory method for SignatureDataTransform.
+        internal static IDataTransform Create(IHostEnvironment env, Arguments args, IDataView input)
         {
-            Host.AssertValueOrNull(ch);
-            Host.AssertValue(input);
-            Host.Assert(0 <= iinfo && iinfo < Infos.Length);
-            Host.Assert(Infos[iinfo].TypeSrc.IsVector & Infos[iinfo].TypeSrc.ItemType.IsText);
-            disposer = null;
+            Contracts.CheckValue(env, nameof(env));
+            env.CheckValue(args, nameof(args));
+            env.CheckValue(input, nameof(input));
 
-            var ex = _exes[iinfo];
-            Language stopWordslang = ex.Lang;
-            var lang = default(ReadOnlyMemory<char>);
-            var getLang = ex.LangsColIndex >= 0 ? input.GetGetter<ReadOnlyMemory<char>>(ex.LangsColIndex) : null;
+            env.CheckValue(args.Column, nameof(args.Column));
+            var cols = new ColumnInfo[args.Column.Length];
+            for (int i = 0; i < cols.Length; i++)
+            {
+                var item = args.Column[i];
+                cols[i] = new ColumnInfo(item.Source ?? item.Name,
+                   item.Name,
+                   item.Language ?? args.Language,
+                   item.LanguagesColumn ?? args.LanguagesColumn);
+            }
+            return new StopWordsRemoverTransform(env, cols).MakeDataTransform(input);
+        }
 
-            var getSrc = GetSrcGetter<VBuffer<ReadOnlyMemory<char>>>(input, iinfo);
-            var src = default(VBuffer<ReadOnlyMemory<char>>);
-            var buffer = new StringBuilder();
-            var list = new List<ReadOnlyMemory<char>>();
+        // Factory method for SignatureLoadDataTransform.
+        private static IDataTransform Create(IHostEnvironment env, ModelLoadContext ctx, IDataView input)
+            => Create(env, ctx).MakeDataTransform(input);
 
-            ValueGetter<VBuffer<ReadOnlyMemory<char>>> del =
-                (ref VBuffer<ReadOnlyMemory<char>> dst) =>
-                {
-                    var langToUse = stopWordslang;
-                    UpdateLanguage(ref langToUse, getLang, ref lang);
+        // Factory method for SignatureLoadRowMapper.
+        private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema inputSchema)
+            => Create(env, ctx).MakeRowMapper(inputSchema);
 
-                    getSrc(ref src);
-                    list.Clear();
+        protected override IRowMapper MakeRowMapper(ISchema schema) => new Mapper(this, Schema.Create(schema));
```

Comment:
>ISchema [](start = 52, length = 7)

At some point we should change the base class's abstract method to be `IRowMapper MakeRowMapper(Schema schema)`